app/src/components/GrantCard.vue

```diff
@@ -51,14 +51,14 @@ import { formatNumber, formatAddress, getEtherscanUrl, pushRoute } from 'src/uti
 // --- Icons ---
 import { Cart2Icon as CartIcon } from '@fusion-icons/vue/interface';
 
-function getTotalRaised(grantId: any) {
+function getTotalRaised(grantId: BigNumberish) {
   const { grantRounds, grantContributions } = useDataStore();
-  const contributions = filterContributionsByGrantId(grantId.value.toString(), grantContributions?.value || []);
+  const contributions = filterContributionsByGrantId(grantId.toString(), grantContributions?.value || []);
```

Comment:
[`BigNumberish`](https://github.com/ethers-io/ethers.js/blob/4166b2753d0d6b230a3a22b5a2b6c4e1906a7af7/packages/bignumber/src.ts/bignumber.ts#L25) can be of type `Bytes` and I'm not sure if that has a `toString()` method. Might want to use this to be safe:

```suggestion
  const contributions = filterContributionsByGrantId(BigNumber.from(grantId).toString(), grantContributions?.value || []);
```

Reply:
I've changed grantId to be a number everywhere 👍